src/ui/language_region_notify_popup.ui

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="Adw" version="1.0"/>
+  <template class="LanguageRegionNotifyPopup" parent="AdwAlertDialog">
```

Comment:
The current status of developing system-wide styles are not clear. If you can spot the light on this matter - it would be perfect. But now this notification looks not aligned with app design.

Reply:
Ok, will change to GtkWindow base approach for now. I'll keep the adwaita stuff in a branch for future consideration.